projects/testing/cypress/snapshot/command.ts

```diff
@@ -1,5 +1,6 @@
 /// <reference types="cypress" />
-
+// eslint-disable-next-line @taiga-ui/no-deep-imports
+import {tuiIsObject, tuiIsString} from '@taiga-ui/cdk/utils/miscellaneous';
```

Comment:
Why deep import ?

I think Ivy can tree-shake it.

Reply:
<img width="1206" alt="image" src="https://user-images.githubusercontent.com/12021443/213391793-048f2ee8-43c4-4152-b844-ec5f50f46934.png">